decidim-core/spec/validators/uploader_content_type_validator_spec.rb

```diff
@@ -41,9 +41,11 @@ def self.model_name
 
       validates_upload(:file, **validation_options.merge(uploader: mount_class))
 
+      # rubocop:disable RSpec/FactoryBot/SyntaxMethods
```

Comment:
Maybe we could also do something like?
```ruby
let(:dummy_record) { validatable.new }
let(:organization) { create(:organization) }

before do
  allow(dummy_record).to receive(:organization).and_return(organization)
end
```

Reply:
This is not working ... It will fail with: RSpec/SubjectStub: Do not stub methods of the object under test.

Reply:
Ah of course, sorry about that. I think your solution is much better!